common/src/main/java/tachyon/network/protocol/RPCMessage.java

```diff
@@ -31,7 +31,10 @@
 
   // The possible types of RPC messages.
   public enum Type implements EncodedMessage {
-    RPC_BLOCK_REQUEST(0), RPC_BLOCK_RESPONSE(1);
+    RPC_BLOCK_REQUEST(0),
```

Comment:
NIT: should this better be `RPC_BLOCK_READ_REQUEST` and `RPC_BLOCK_RESPONSE`? Maybe in another PR